src/Agent.Plugins/BuildArtifact/BuildArtifactPluginV1.cs

```diff
@@ -73,6 +74,7 @@ public class DownloadBuildArtifactTaskV1_0_0 : BuildArtifactTaskPluginBaseV1
         static readonly string buildVersionToDownloadLatest = "latest";
         static readonly string buildVersionToDownloadSpecific = "specific";
         static readonly string buildVersionToDownloadLatestFromBranch = "latestFromBranch";
+        static readonly string extractedTarsTempDir = "extracted_tars";
```

Comment:
Why do we need this temp dir?

Reply:
This is how it worked in the V0 version of this task.
All downloaded .tar files are extracted to $(Agent.TempDir)/extracted_tars, and then copied over to target directory of this task.